server/src/session.ts

```diff
@@ -134,10 +134,11 @@ export class Session {
         continue;
       }
       const project = this.projectService.getDefaultProjectForScriptInfo(scriptInfo);
-      if (!project || !project.languageServiceEnabled) {
+      const ngLS = project ?.getLanguageService();
```

Comment:
`getLanguageService` will not return undefined if the language service is disabled, so the check for `languageServiceEnabled` is still needed.

Reply:
https://github.com/microsoft/TypeScript/blob/bc1e7728df93cfbd70ab2966b37952d5978ba677/src/server/project.ts#L549-L553